Missing server_role in sample deployment.toml config files in integration tests

Missing server_role in sample deployment.toml config files in integration tests casing test failures. This deployment.toml file is is to start the server in Tests.

product-apim/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/originalFile/deployment.toml

**Test Failied**
```java
[ERROR] testJWTTokenAPITestCase(org.wso2.am.integration.tests.token.TokenAPITestCase)  Time elapsed: 1,903.925 s  <<< FAILURE!
org.wso2.am.integration.test.utils.APIManagerIntegrationTestException: Key generation fails. Error: getting application Id failed. Error: A JSONObject text must begin with '{' at character 0
	at org.wso2.am.integration.tests.token.TokenAPITestCase.testJWTTokenAPITestCase(TokenAPITestCase.java:311)
Caused by: org.wso2.am.integration.test.utils.APIManagerIntegrationTestException: getting application Id failed. Error: A JSONObject text must begin with '{' at character 0
	at org.wso2.am.integration.tests.token.TokenAPITestCase.testJWTTokenAPITestCase(TokenAPITestCase.java:311)
Caused by: org.json.JSONException: A JSONObject text must begin with '{' at character 0
	at org.wso2.am.integration.tests.token.TokenAPITestCase.testJWTTokenAPITestCase(TokenAPITestCase.java:311)
```

--- a/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/applicationSharing/deployment.toml
+++ b/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/applicationSharing/deployment.toml
@@ -4,6 +4,7 @@ node_ip = "127.0.0.1"
 #offset=0
 mode = "single" #single or ha
 base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
+server_role = "default"
 
 [super_admin]
 username = "admin"

--- a/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/common/deployment.toml
+++ b/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/common/deployment.toml
@@ -4,6 +4,7 @@ node_ip = "127.0.0.1"
 #offset=0
 mode = "single" #single or ha
 base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
+server_role = "default"
 
 [super_admin]
 username = "admin"

--- a/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/corsACACTest/deployment.toml
+++ b/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/corsACACTest/deployment.toml
@@ -4,6 +4,7 @@ node_ip = "127.0.0.1"
 #offset=0
 mode = "single" #single or ha
 base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
+server_role = "default"
 
 [super_admin]
 username = "admin"

--- a/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/corsACACTest/original/deployment.toml
+++ b/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/corsACACTest/original/deployment.toml
@@ -4,6 +4,7 @@ node_ip = "127.0.0.1"
 #offset=0
 mode = "single" #single or ha
 base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
+server_role = "default"
 
 [super_admin]
 username = "admin"

--- a/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/customHeaderTest/deployment.toml
+++ b/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/customHeaderTest/deployment.toml
@@ -2,6 +2,7 @@
 node_ip = "127.0.0.1"
 mode = "single" #single or ha
 base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
+server_role = "default"
 
 [super_admin]
 username = "admin"

--- a/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/emailusernametest/deployment.toml
+++ b/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/emailusernametest/deployment.toml
@@ -4,6 +4,7 @@ node_ip = "127.0.0.1"
 #offset=0
 mode = "single" #single or ha
 base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
+server_role = "default"
 
 [super_admin]
 username = "admin"

--- a/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/tokenTest/deployment.toml
+++ b/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/tokenTest/deployment.toml
@@ -4,6 +4,7 @@ node_ip = "127.0.0.1"
 #offset=0
 mode = "single" #single or ha
 base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
+server_role = "default"
 
 [super_admin]
 username = "admin"

--- a/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/tokenTest/urlSafeTokenTest/deployment.toml
+++ b/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/tokenTest/urlSafeTokenTest/deployment.toml
@@ -4,6 +4,7 @@ node_ip = "127.0.0.1"
 #offset=0
 mode = "single" #single or ha
 base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
+server_role = "default"
 
 [super_admin]
 username = "admin"

--- a/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/webSocketTest/deployment.toml
+++ b/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/webSocketTest/deployment.toml
@@ -4,6 +4,7 @@ node_ip = "127.0.0.1"
 #offset=0
 mode = "single" #single or ha
 base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
+server_role = "default"
 
 [super_admin]
 username = "admin"

--- a/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/lifecycletest/mutualssl/deployment.toml
+++ b/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/lifecycletest/mutualssl/deployment.toml
@@ -4,6 +4,7 @@ node_ip = "127.0.0.1"
 #offset=0
 mode = "single" #single or ha
 base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
+server_role = "default"
 
 [super_admin]
 username = "admin"

--- a/modules/integration/tests-integration/tests-config/src/test/resources/artifacts/AM/fullConfigurations/scenario1/deployment.toml
+++ b/modules/integration/tests-integration/tests-config/src/test/resources/artifacts/AM/fullConfigurations/scenario1/deployment.toml
@@ -4,6 +4,7 @@ node_ip = "127.0.0.1"
 #offset=0
 mode = "single" #single or ha
 base_path = "localhost:9443"
+server_role = "default"
 
 [super_admin]
 username = "admin"

--- a/modules/integration/tests-integration/tests-config/src/test/resources/artifacts/AM/individualConfigurations/scenario1/deployment.toml
+++ b/modules/integration/tests-integration/tests-config/src/test/resources/artifacts/AM/individualConfigurations/scenario1/deployment.toml
@@ -4,6 +4,7 @@ node_ip = "127.0.0.1"
 #offset=0
 mode = "single" #single or ha
 base_path = "localhost:9443"
+server_role = "default"
 
 [super_admin]
 username = "admin"

